composer.json

```diff
@@ -26,11 +26,15 @@
     "require-dev": {
         "amphp/phpunit-util": "^1.0",
         "friendsofphp/php-cs-fixer": "^2.9",
-        "phpunit/phpunit": "^6.5"
+        "phpunit/phpunit": "^6.5",
+        "php-coveralls/php-coveralls": "^1.0"
     },
     "config": {
         "platform": {
             "php": "7.0.0"
         }
+    },
+    "scripts": {
+        "fix-syntax": "php vendor/bin/php-cs-fixer --diff --dry-run -v fix"
```

Comment:
If we want to add that, it should probably not use `--dry-run`. Does Composer allow adding arguments so that `composer fix-syntax` and `composer fix-syntax --dry-run` work? I haven't really used Composer scripts, yet.

Reply:
It cannot. The Composer cannot add additional arguments.
And the ```composer fix-syntax --dry-run``` does not work.

Reply:
Ok, it can add additional argument.
Just use ```composer fix-syntax -- --dry-run```.